src/codelabs/iterables.md

```diff
@@ -250,18 +250,6 @@ There is no right or wrong way.
 Use the way that works best for you, 
 and that makes your code easier to read and understand.
 
-In the example, `firstWhereWithOrElse()` calls `firstWhere()` with
```

Comment:
I assume you're removing this because `firstWhereWithOrElse` doesn't exist in the code above, correct? Nice catch!

Despite that, it seems this text is still useful to explain the final example in the code block above. 

Perhaps instead of referencing `firstWhereWithOrElse` which doesn't exist, we could still refer to that example somehow. Perhaps:

```
The final use of `firstWhere` above specifies
....
```

Let me know what you think. Thanks again!

Reply:
It exists the issue is I have accidentally committed using my office git hub account and when pushed branched used personal account